scripts/generateAssetData/utils/portals.ts

```diff
@@ -1,31 +1,66 @@
 import type { ChainId } from '@shapeshiftoss/caip'
 import { ASSET_NAMESPACE, bscChainId, toAssetId } from '@shapeshiftoss/caip'
-import type { Asset } from '@shapeshiftoss/types'
+import type { Asset, AssetsByIdPartial } from '@shapeshiftoss/types'
 import axios from 'axios'
 import qs from 'qs'
 import { getAddress, isAddressEqual, zeroAddress } from 'viem'
 import { CHAIN_ID_TO_PORTALS_NETWORK } from 'lib/market-service/portals/constants'
-import type { GetTokensResponse, TokenInfo } from 'lib/market-service/portals/types'
+import type {
+  GetPlatformsResponse,
+  GetTokensResponse,
+  PlatformsById,
+  TokenInfo,
+} from 'lib/market-service/portals/types'
+import { isSome } from 'lib/utils'
 
+import generatedAssetData from '../../../src/lib/asset-service/service/generatedAssetData.json'
 import { colorMap } from '../colorMap'
 import { createThrottle } from '.'
 
-export const fetchPortalsTokens = async (
+const assets = generatedAssetData as unknown as AssetsByIdPartial
+
+const PORTALS_API_KEY = process.env.REACT_APP_PORTALS_API_KEY
+if (!PORTALS_API_KEY) throw new Error('REACT_APP_PORTALS_API_KEY not set')
+
+const fetchPortalsPlatforms = async (): Promise<PlatformsById> => {
+  const url = 'https://api.portals.fi/v2/platforms'
```

Comment:
suggestion: use an env variable

Reply:
45660caa58 and confirmed happy 
<img width="689" alt="image" src="https://github.com/user-attachments/assets/95d25aa0-b20d-4445-9330-88868d51b155">